modules/ROOT/pages/design-duplicate-project.adoc

```diff
@@ -0,0 +1,29 @@
+= Duplicate a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+You can duplicate projects in API Designer. You might want to duplicate a project if you need to make extensive changes to the project without losing the original on which the duplicate project is based.
+
+Any dependencies that are listed in the original project are retained in the duplicate project.
+
+== Before You Begin
+
+* Ensure that the mocking service is not turned on.
+* Ensure that the project is not in read-only mode.
+
+== Procedure
+
+. Open the project.
+. Click the gear in the top-right corner of the editor.
+. Select *Duplicate*.
+. Name the duplicate project and click *Duplicate*.
+
+== Result
+
+API Designer opens a new browser tab that displays the new project. If the original project is in the text editor, the duplicate project is opened in the text editor. If the original project is in the visual editor, the duplicate project is opened in the visual editor.
+
+If the original was in a sub-branch, the duplicate is on the master branch.
```

Comment:
If the project was in a sub-branch the duplicate is on that branch, not on master.